recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -14,5 +14,10 @@ conda install -n root --yes --quiet jinja2 conda-build anaconda-client
 :: KLUDGE to work around changes in conda-build 2.0.0
 conda install -n root --yes --quiet conda-build=1
 
+:: Needed for building extensions in python2.7 x64 with cmake.
+:: Since python version and arch is not known at this point, install it everywhere.
+conda install -n root --yes --quiet vs2008_express_vc_python_patch
+call setup_x64
```

Comment:
Also you can see some testing of this in PR ( https://github.com/conda-forge/staged-recipes/pull/1737 ) where we ran Eigen's test suite, which is a known case that requires this patch (e.g. a CMake build using Visual Studio generators particularly for VS 2008 64-bit). Admittedly there we hacked it into `appveyor.yml`, but the idea should remain the same here.